### ISSUE TITLE
No bounding boxes on test images

### ISSUE THREAD

**Issue by Balavamsi1999** [AntonMu/TrainYourOwnYOLO#127]
I trained the model and the loss was around 1.0 but during testing, suppose if I gave trained image also, it is not predicting the label and not drawing bounding box and in detection_Results.csv also no rows are inserted please help…
I tried to debug:
its printing found 0 boxes on image in the terminal
so I tried to print predictions
prediction in yolo.py was [ ] which results in
prediction in utils.py was [ ] which results in
prediction in Detector.py was [ ]

Looks like error is occurring here in    **2_Training/src/keras_yolo3/yolo.py** : 

out_boxes, out_scores, out_classes = self.sess.run(
            [self.boxes, self.scores, self.classes],
            feed_dict={
                self.yolo_model.input: image_data,
                self.input_image_shape: [image.size[1], image.size[0]],
                K.learning_phase(): 0
            },
        )
print(out_boxes) # prints [ ]

where: 
self.boxes= Tensor("concat_11:0", shape=(?, 4), dtype=float32)
self.scores=Tensor("concat_12:0", shape=(?,), dtype=float32)
self.classes=Tensor("concat_13:0", shape=(?,), dtype=int32)

is there any issue in self.boxes / self.scores/ self.classes ??

please help @AntonMu 

I didn't use voot for extracting object from image instead I used thresholding as background in images were white and created vott-csv-export and added Annotations-export.csv file in it which contains image path,bounding box co-ordinates (x min,y min,x max,y max) and label.

**Comment by Balavamsi1999** [comment]
If you dont give enough epochs we get this problem, so to solve this issue increase the no:of epochs...
